tests/unit/OptionsListUtilsTest.js

```diff
@@ -597,23 +619,41 @@ describe('OptionsListUtils', () => {
         () => Report.setReportWithDraft(1, true)
             .then(() => {
                 // When we call getSidebarOptions() with no search value
-                const results = OptionsListUtils.getSidebarOptions(REPORTS, PERSONAL_DETAILS, 0, CONST.PRIORITY_MODE.GSD);
+                const results = OptionsListUtils.getSidebarOptions(REPORTS_WITH_MORE_PINS, PERSONAL_DETAILS, 0, CONST.PRIORITY_MODE.GSD);
 
                 // Then expect all of the reports to be shown both multiple and single participant except the
                 // report that has no lastMessageTimestamp and the chat with Thor who's message is read
-                expect(results.recentReports.length).toBe(_.size(REPORTS) - 2);
+                expect(results.recentReports.length).toBe(_.size(REPORTS_WITH_MORE_PINS) - 2);
 
                 // That no personalDetails are shown
                 expect(results.personalDetails.length).toBe(0);
 
-                // And Mister Fantastic is alphabetically the fourth report and has an unread message
-                // despite being pinned
-                expect(results.recentReports[4].login).toBe('reedrichards@expensify.com');
 
-                // And Black Panther is alphabetically the first report and has an unread message
-                expect(results.recentReports[0].login).toBe('tchalla@expensify.com');
```

Comment:
Could you please add tests for unread reports?

Reply:
[3] [4] [5] is unread reports.

Reply:
Ok. then I think we need to indicate that in the comments as well.